src/components/ChatRoom.tsx

```diff
@@ -32,11 +32,8 @@ export default function Room({ menteeId }: {
   return !room ? <Loader /> :
     <VStack spacing={paragraphSpacing * 1.5} align="start" maxWidth="800px">
       <MessageCreator roomId={room.id} />
-
-      {room.messages.sort((a, b) => moment(a.updatedAt)
-        .isAfter(moment(b.updatedAt)) ? -1 : 1)
-        .map(m => <Message key={m.id} message={m} />)
-      }
+      {room.messages.sort((a, b) => compareDate(a.updatedAt, b.updatedAt))
```

Comment:
actually I should have used `createdAt`. can you help the fix in this or a separate PR?